src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -482,7 +482,7 @@
   <Target Name="SetRazorCodeAnalysisProperties" BeforeTargets="RazorCoreCompile" AfterTargets="CopyFilesToOutputDirectory">
     <PropertyGroup Condition=" $(ErrorLog) == $(SonarCompileErrorLog) ">
       <!-- Keep the path so that we could set SonarQubeSetting with sonar.language.roslyn.reportFilePaths -->
-      <RazorSonarCompileErrorLog>$(TargetDir)$(TargetName)$(RazorTargetNameSuffix)$(TargetExt).RoslynCA.json</RazorSonarCompileErrorLog>
+      <RazorSonarCompileErrorLog>$(RazorSonarCompileErrorLog)</RazorSonarCompileErrorLog>
```

Comment:
Is there a way to add integration test for it on scanner side? This one was hard to spot.
I added similar IT on sonar-dotnet side where we have custom targets file. https://github.com/SonarSource/sonar-dotnet/pull/4016
See changes in `analyzers/its/sources/NetCore31` directory.

We should have test for .NET Core 2.1 project to test the old functionality (if not already present), test for .NET Core 3.1 or .NET 5 to test current functionality and one without `global.json` to test the latest SDK available that will blow in case it changes again.

The minimal project should target SDK `Microsoft.NET.Sdk.Web` and link SonarLint.xml to analyze generated, have empty `Main` with no other code and contain single `cshtml` file.